scripting/cwx/item_config.sp

```diff
@@ -305,10 +305,53 @@ int EquipCustomItem(int client, const CustomItemDefinition item) {
 			SetEntProp(itemEntity, Prop_Send, "m_iAccountID", accountid);
 		}
 	}
-	
-	// TODO: implement a version that nullifies runtime attributes to their defaults
-	SetEntProp(itemEntity, Prop_Send, "m_bOnlyIterateItemViewAttributes",
-			!item.bKeepStaticAttributes);
+
+	if(!item.bKeepStaticAttributes) {
+		TF2Attrib_RemoveAll(itemEntity);
+
+		char valueType[2];
+		char valueFormat[64];
+
+		int staticAttribs[16];
+		float staticAttribsValues[16];
+
+		TF2Attrib_GetStaticAttribs(item.defindex, staticAttribs, staticAttribsValues);
+
+		for(int i = 0; i < 16; i++)
+		{
+			if(staticAttribs[i] == 0) {
+				continue;
+			}
+
+			// Probably overkill
+			if(staticAttribs[i] == 796 || staticAttribs[i] == 724 || staticAttribs[i] == 817 || staticAttribs[i] == 834 
```

Comment:
If we can get a working implementation without hardcoded attributes, that'd be good.  This kind of thing is brittle and not something I want to think about maintaining in the future.

I think the main ones would be `paintkit_proto_def_index` (834) and `has team color paintkit` (745); the other ones have empty description formats so I'd imagine they're not an issue.

Reply:
Scratch that; the attribute listing page lied to me.  The other ones do have `value_is_additive`.

Reply:
So should I remove them? Although hardcoding some attributes might not be the prettiest thing to do ( which is fair ), I did it to skip some unnecessary cycles and avoid some things breaking ( e.g. `817 inspect_viewmodel_offset`, I don't know if nullifying would actually break some "QoL" aspects of the weapons so I put them just in case ).

Reply:
I'd say remove them, but at the same time having other things be potentially broken isn't great.  This was also one of the things I wasn't satisfied with when I was brainstorming the thing myself.

Reply:
That's fair. However, the amount of attributes is so small that it's negligible. It'd be nice to have an automatic way though.

Reply:
Addressed this in the newest commit.